sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -884,7 +885,8 @@ class DataSourceV2SQLSuite
             .isEmpty, s"$key is a reserved namespace property and ignored")
           val meta =
             catalog("testcat").asNamespaceCatalog.loadNamespaceMetadata(Array("reservedTest"))
-          assert(meta.get(key) === null, "reserved properties should not have side effects")
+          assert(!Option(meta.get(key)).exists(_.contains("foo")),
```

Comment:
what's wrong with `meta.get(key) === null`?

Reply:
this can be fixed if we revert the change in Create syntax

Reply:
is it simply `meta.get(key) != "foo"`?

Reply:
'location' might change the path from relative to absolute

Reply:
`meta.get(key) != "foo"` still works, right?

Reply:
yes, but once if the `location` goes wrong and have side effects ` != 'foo'` is not able to check that.

Reply:
how about `meta.get(key) == null || !meta.get(key).contains("foo")`?

Reply:
OK